grid2op/Action/BaseAction.py

```diff
@@ -369,6 +369,8 @@ def __eq__(self, other) -> bool:
             Whether the actions are equal or not.
 
         """
+        if other is None:
+            return not self.as_dict()
```

Comment:
Nope, any test to "==None" should return False, by convention.

None is not equal to "do nothing"

Reply:
done